ESPixelStick/ESPixelStick.ino

```diff
@@ -502,29 +508,41 @@ void loop()
     // Reboot handler
     if (NotRebootingValue != RebootCount)
     {
-        if(0 == --RebootCount)
+        if (0 == --RebootCount)
         {
-            logcon (String(CN_stars) + CN_minussigns + F ("Internal Reboot Requested. Rebooting Now"));
-            delay (REBOOT_DELAY);
-            ESP.restart ();
+            logcon(String(CN_stars) + CN_minussigns + F("Internal Reboot Requested. Rebooting Now"));
+            delay(REBOOT_DELAY);
+            ESP.restart();
         }
     }
 
     if (NO_CONFIG_NEEDED != ConfigLoadNeeded)
     {
-        if(abs(now() - ConfigLoadNeeded) > LOAD_CONFIG_DELAY)
+        if (abs(now() - ConfigLoadNeeded) > LOAD_CONFIG_DELAY)
         {
-            FeedWDT ();
-            LoadConfig ();
+            FeedWDT();
+            LoadConfig();
         }
     }
 
     if (ConfigSaveNeeded)
     {
-        FeedWDT ();
-        SaveConfig ();
+        FeedWDT();
+        SaveConfig();
+    }
+    if(millis() >= time_now + period){
```

Comment:
Move this call to the oled files and make a call that is conditionally compiled.